bom/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.quarkiverse.googlecloudservices</groupId>
+    <artifactId>quarkus-google-cloud-services-bom</artifactId>
+    <name>Quarkus - Google Cloud Services - BOM</name>
+    <version>0.6.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <google-cloud-sdk.version>16.1.0</google-cloud-sdk.version>
+        <google-cloud-graalvm.version>0.3.0</google-cloud-graalvm.version>
+        <!-- The Google Cloud BOM includes the 'android' version of Guava so we need to fix the 'jre' version.
+        Make sure to keep these two libraries compatible. -->
+        <guava.version>30.0-jre</guava.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Google Cloud dependencies, imported as a BOM -->
```

Comment:
I am not sure you need to add these dependencies here, as they are resolved transitively, but @aloubyansky might know better. The BOM should contain only your extension´s runtime and deployment modules AFAIK

Reply:
So you suggest to keep them inside my parent pom ?

The Quarkus BOM contains all dependencies that's why I add the managed version here.

Reply:
> So you suggest to keep them inside my parent pom ?
Yes, I think so.

> The Quarkus BOM contains all dependencies that's why I add the managed version here.
Gotcha, I think once the BOM is flattened using @aloubyansky´s plugin the result will be the same, but let´s wait for his take on this

Reply:
Flattening does not introduce new dependency constraints. It flattens the parent-child hierarchy.
@loicmathieu your bom is supposed to include the artifacts your project produces plus dependency constraints for other artifacts that your artifacts require to function properly.